docs/reference/ml/trained-models/apis/put-trained-models.asciidoc

```diff
@@ -31,8 +31,8 @@ Requires the `manage_ml` cluster privilege. This privilege is included in the
 == {api-description-title}
 
 The create trained model API enables you to supply a trained model that is not
-created by {dfanalytics}.
-
+created by {dfanalytics}. The trained model must be in a TorchScript
+representation for use with {stack} {ml} features.
```

Comment:
Does it make sense to maybe just refer people to the guide? It's a lot unsaid to just mention "TorchScript representation".

```suggestion
created by {dfanalytics}. The trained model must be a supported PyTorch NLP model for use with {stack} {ml} features. Please see xxxx for more details.
```

Reply:
I believe the user could also supply a non PyTorch model to this API. It could be a boosted tree the user trained outside the stack and then converted into the right format.

Reply:
Good point!  I've added a link to the (in-progress) NLP docs but only where we're talking about the specific model type.